src/status_im/multiaccounts/login/core.cljs

```diff
@@ -150,6 +168,27 @@
         update-address
         (update :accounts (partial mapv update-address)))))
 
+(re-frame/reg-fx
+ ;;TODO: this could be replaced by a single API call on status-go side
```

Comment:
why isn't it replaced?

Reply:
For now we stick to simple RPC Valls, we'll improve that next in version